Source/WebCore/Modules/modern-media-controls/media/scrubbing-support.js

```diff
@@ -68,11 +68,22 @@ class ScrubbingSupport extends MediaControllerSupport
         if (isNaN(media.duration))
             return;
 
-        let buffered = 0;
-        for (let i = 0, count = media.buffered.length; i < count; ++i)
-            buffered = Math.max(media.buffered.end(i), buffered);
+        // Let's find the buffered time range containing the current time and
+        // set the scrubber's secondary value to the end of that range.
+        const endTimeForBufferedRangeContainingCurrentTime = () => {
+            const ranges = media.buffered;
+            const currentTime = media.currentTime;
+            for (let i = 0, count = ranges.length; i < count; ++i) {
```

Comment:
Yeah, I'm not sure why it was written that way originally. Probably to avoid hitting IDL for the `length` property again, but that seems pretty pointless. Will fix in commit.